cmd/kops/root.go

```diff
@@ -126,9 +126,10 @@ func NewCmdRoot(f *util.Factory, out io.Writer) *cobra.Command {
 	if strings.HasSuffix(defaultStateStore, "/") {
 		defaultStateStore = strings.TrimSuffix(defaultStateStore, "/")
 	}
-	cmd.PersistentFlags().StringVarP(&rootCommand.RegistryPath, "state", "", defaultStateStore, "Location of state storage")
+	cmd.PersistentFlags().StringVarP(&rootCommand.RegistryPath, "state", "", defaultStateStore, "Location of state storage. Overrides $KOPS_STATE_STORE")
 
-	cmd.PersistentFlags().StringVarP(&rootCommand.clusterName, "name", "", "", "Name of cluster")
+	defaultClusterName := os.Getenv("KOPS_CLUSTER_NAME")
+	cmd.PersistentFlags().StringVarP(&rootCommand.clusterName, "name", "", defaultClusterName, "Name of cluster. Overrides $KOPS_CLUSTER_NAME")
```

Comment:
Just a nit pick, since we will soon have windows support

> Overrides $KOPS_CLUSTER_NAME

Should we do

> Overrides KOPS_CLUSTER_NAME environment variable

Reply:
How does that look/sound on both environment variables?